src/common/types/api-error-type.ts

```diff
@@ -0,0 +1,17 @@
+export type ApiSuccessResponse<T> = {
+  ok: true;
+  data: T;
+};
+
+export type ApiError = {
+  code: string;
```

Comment:
P1;
`src/common/types/error-code.ts` 코드를  만드셨는데, 오타로 인한 버그 방지를 위해 사용하는 것은 어떨까요?

```suggestion
  code: ErrorCode;
```

Reply:
동의합니다. 

오타 방지 위해 ApiError.code 타입을 string 대신 ErrorCode로 좁히는 방향으로 정리하겠습니다. (필요 시 확장 가능한 구조로 유지)